pkg/resources/external_function_acceptance_test.go

```diff
@@ -391,6 +391,53 @@ func TestAcc_ExternalFunction_issue2528(t *testing.T) {
 	})
 }
 
+// Proves that header parsing handles values wrapped in curly braces, e.g. `value = "{1}"`
+func TestAcc_ExternalFunction_HeaderParsing(t *testing.T) {
```

Comment:
is this a result of a particular GH issue? If so, let's at least link it in the PR description

Reply:
It was an internal request (no GH issue was created).